src/Util/ApiDocumentationBuilderInterface.php

```diff
@@ -9,17 +9,17 @@
  * file that was distributed with this source code.
  */
 
-namespace ApiPlatform\Core\Hydra;
+namespace ApiPlatform\Core\Util;
```

Comment:
OK why not keeping this command interface, but:

* Should be move to a `Documentation` namespace
* Can be used to have a common controller for both docs (and an optional `?format=swagger|hydra` for instance)